tools/chapel-py/src/PythonWrapper.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2023-2025 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef CHAPEL_PY_PYTHON_WRAPPER_H
+#define CHAPEL_PY_PYTHON_WRAPPER_H
```

Comment:
Please add a comment for this header to explain why it is needed (to ensure `Python.h` is always included with the necessary `#define`s set).